src/Cohesity.Powershell/Scripts/Utility/Set-CohesityCmdletConfig.ps1

```diff
@@ -27,7 +28,10 @@ function Set-CohesityCmdletConfig {
         $LogResponseData = $false,
         [Parameter(Mandatory = $false, ParameterSetName = 'LogHeaderDetail')]
         [ValidateSet($true, $false)]
-        $LogHeaderDetail = $false
+        $LogHeaderDetail = $false,
+        [Parameter(Mandatory = $false, ParameterSetName = 'RefreshToken')]
```

Comment:
Make sure the documentation is updated and usage and examples are clearly mentioned

Reply:
Will take care of it